public/js/migrate.js

```diff
@@ -75,6 +75,11 @@
         var _this = event.data.self;
         var $target = $(event.target);
 
+        if (! _this.icinga.hasModule('monitoring')) {
+            // No need to render the popup if monitoring isn't loaded
```

Comment:
I'd go even so far to say: Why loading this JS if the module isn't active? :grin:

Reply:
However, the popup will never be rendered, even though monitoring is active! Because it can be the case that `icingadb` is loaded first, that's why I did it with `hasModule`.

Reply:
> And that's the only thing you're interested in, right?

Exactly!!